CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 1.4.1
```

Comment:
You've completed steps 3 and 4 but you missed 1 and 2: https://github.com/a-scie/jump/blob/main/RELEASE.md

Reply:
I didn't know if you wanted me to do all the changes, but I'll do that!

Reply:
Done.

Reply:
Thanks. This way I can tag & push & get a release out ~presently.